packages/react/src/Hero/Hero.tsx

```diff
@@ -1,69 +1,94 @@
-import React, {forwardRef} from 'react'
+import React, {forwardRef, PropsWithChildren, useMemo} from 'react'
 import clsx from 'clsx'
 import styles from './Hero.module.css'
-import {Button} from '../Button'
-import {Heading} from '../Heading'
+import {Button, ButtonBaseProps} from '../Button'
+import {Heading, HeadingProps} from '../Heading'
 import {Text} from '../Text'
 
 import type {BaseProps} from '../component-helpers'
 import '@primer/brand-primitives/lib/design-tokens/css/tokens/functional/components/hero/base.css'
 
-type Action = {
-  text: string
+export type HeroProps = BaseProps<HTMLDivElement> & {
```

Comment:
as this is a `<section>` wrapper, should we use `HTMLElement` instead?

Reply:
Good catch! Swapped this for `HTMLElement`.